### PR TITLE
Small changes

### PR DESCRIPTION
**Pull request by WKocur**

### CODE REVIEW

File: app/controllers/admin/users_controller.rb [hunk 1]
```diff
@@ -10,6 +10,7 @@ def index
 
   # GET /users/1
   def show
+    @displayed_attributes = @user.attributes.keys.select{|k| k != 'encrypted_password'}
```
Comment (lucacug) [inline app/controllers/admin/users_controller.rb]: Here you can use `reject` instead of inverting the `select`
`@user.attributes.keys.reject{ |k| k == 'encrypted_password' }`

File: app/views/surveys/_organisational_fields.html.erb [hunk 2]
```diff
@@ -285,7 +285,7 @@
 
   <p>
     Organisation funding sources:
-    <% sources = survey.settings["organisation_funding_sources"].map{|k| surveys_data("organisation_funding_sources")[k.to_sym]}.compact! %>
```
Comment (lucacug) [inline app/views/surveys/_organisational_fields.html.erb]: Could you explain why did you remove the `to_sym` bit, it was working well for me before I think.

Comment (WKocur) [reply app/views/surveys/_organisational_fields.html.erb]: Because it wasn't working for me... It was returning nil.
